### ISSUE TITLE
server generate : can't run custom template in operationGroups

### ISSUE THREAD

**Issue by fredbi** [go-swagger/go-swagger#1290]
## Problem statement
Defined a custom template.
Declared it in the config YAML section 'operation_groups:'
Run server generate on this config

Nothing happens! The generator found an operation group but didn't take template into account.
Retesting with a wrong source template name indicates that there is not even an attempt to load the template.

Will fix this with a P.R as soon as the regression on security items allows a CI based on current master to run...

Cheers,

Fred

## Environment
swagger version: 0.12
go version: 9.1
OS:   Linux

**Comment by fredbi** [comment]
Merged with 1290-fix server generation now supports operationGroups templates jus… #1298

**Comment by ZachEddy** [comment]
Hi there! I also stumbled across this issue using Homebrew.

```bash
# after following homebrew instructions and freshly installing go-swagger
swagger version
version: 0.13.0
commit: 8135eb6728e43b73489e80f94426e6d387809502
```

I had problems with `operationGroups` and using custom templates. However, using `go-get` to install go-swagger instead of Homebrew fixed the issue.

Not sure if Homebrew uses the latest and most up-to-date version of go-swagger (i.e. I don't know if it has your recent fix), but I thought I'd bring it up just in case.

**Comment by casualjim** [comment]
this fix came in after the 0.13 release but will be included in the next release
